### PR TITLE
Ground truth odometry

### PR DESCRIPTION
**Pull request by amiller27**

### CODE REVIEW

File: src/simulator_adapter.py [hunk 1]
```diff
@@ -228,6 +238,8 @@ def obstacle_odom_callback(msg, topic, data={}):
     accel_pub = rospy.Publisher('acceleration', Vector3Stamped, queue_size=0)
     battery_pub = rospy.Publisher('fc_battery', Float32, queue_size=0)
     status_pub = rospy.Publisher('fc_status', FlightControllerStatus, queue_size=0)
+    if publish_ground_truth_localization:
```
Comment (aftersomemath) [inline src/simulator_adapter.py:241]: These are in the same method. Does it make sense to have two if statements for this? I do recognize that the method is split into publisher and subscriber creation but maybe it would be better to create publishers and subscribers in pairs?

Might even be something for a different PR

**Review by aftersomemath — APPROVED** [review]

File: src/simulator_adapter.py (hunk 1 above)
Comment (amiller27) [reply src/simulator_adapter.py:241]: I organized these by who they're communicating with (publishers to morse, subscribers from morse, publishers to the rest of the world, subscribers from the rest of the world), so don't want to change that in this PR, but this node could definitely be reorganized in the future
